packages/react-components/react-carousel-preview/library/src/components/CarouselSlider/useCarouselSliderStyles.styles.ts

```diff
@@ -0,0 +1,71 @@
+import { makeStyles, mergeClasses } from '@griffel/react';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+import type { CarouselSliderSlots, CarouselSliderState } from './CarouselSlider.types';
+
+export const carouselSliderClassNames: SlotClassNames<CarouselSliderSlots> = {
+  root: 'fui-CarouselSlider',
+  slider: 'fui-CarouselSlider__slider',
+};
+
+// For now, until motion tokens are updated.
+const easingCurve = 'cubic-bezier(0.65, 0, 0.35, 1)';
```

Comment:
FYI these is no such value in the design superset, please reach out to your designer.